enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/ExpressionAcceptanceTest.scala

```diff
@@ -97,15 +97,15 @@ class ExpressionAcceptanceTest extends ExecutionEngineFunSuite with CypherCompar
   test("prepending item to a list should behave correctly in all runtimes") {
     val query = "CYPHER WITH {a:[1,2,3]} AS x RETURN 'a:' + x.a AS r"
 
-    val result = succeedWith(Configs.All - Configs.EnterpriseInterpreted, query)
+    val result = succeedWith(Configs.AllExceptSleipnir, query)
```

Comment:
I wouldn't be against renaming all occurrences of "Sleipnir" || "Registered" in the codebase with one consistent name, i.e., "Slotted". 
This PR already does a fair bit of that (e.g., test names, test scenarios, etc.), so why not do it all now.